### DISCUSSION TITLE
Vercel deployment show js file

### DISCUSSION THREAD

**Discussion by dmitrydrynov** [fastify/fastify#4197] [Q&A]
Hi everyone,
I tried to deploy my fastify project on the Vercel.
I added vercel.json, added api/serverless.js file
After deployment I see source code of the serverless.js file (url https://api-skillkit-441zpjkit-drynov.vercel.app)

On other server I deployment next
yarn build
node server.js

How I can start fastify app on Vercel or how I can set "node server.js" in Vercel?

![image](https://user-images.githubusercontent.com/4089575/184336833-1d5ba879-277c-4119-b198-5906a48f9f0e.png)

**Comment by mcollina** [comment]
Thanks for reporting! Maybe @Ethan-Arrowood could help.

**Comment by dmitrydrynov** [comment]
I tried different options. I think Vercel serverless functions updated and now its return only response data.

As example, this code is working
![image](https://user-images.githubusercontent.com/4089575/184356184-7817ac87-0e67-48b9-b263-04bde9f5d709.png)

**Comment by Ethan-Arrowood** [comment]
I'm assuming you followed this: https://www.fastify.io/docs/latest/Guides/Serverless/#vercel 

Do you have a repository I can look at?
